### ISSUE TITLE
tasks.Retrieval metrics=False. What's the easiest way to then compute Top K metrics after training

### ISSUE THREAD

**Issue by SteWaite** [tensorflow/recommenders#455]
Hi,

Thank you for all the work you do to maintain and develop this library. 

I have metrics=False within the task as the dataset is large. What is the best way to then compute Top k accuracy for the test set?
Just not sure how to modify this line of code -- 

tfrs.metrics.FactorizedTopK(candidates=products.batch(128).map(self.candidate_model)

Thank you

**Comment by SteWaite** [comment]
Found the answer 

https://www.tensorflow.org/recommenders/examples/efficient_serving#evaluating_the_approximation

**Comment by maciejkula** [comment]
Glad you found the answer - please reopen if you need additional assistance.
